lib/surveymonkey/client.rb

```diff
@@ -27,6 +27,11 @@ def respondents(survey_id, options = "")
       get_request("surveys/get_respondent_list", options, body)
     end
 
+    def survey_responses(survey_id, respondent_ids, options = "")
+      body = {:survey_id => survey_id, :respondent_ids => respondent_ids}.to_json
```

Comment:
Can we use `body = { survey_id: survey_id, respondent_ids: respondent_ids }.to_json` or does the request fail like this?